emacs/merlin-imenu.el

```diff
@@ -78,6 +90,12 @@
       (push (cons "Exception" merlin-imenu--exception-list) index))
     (when merlin-imenu--type-list
       (push (cons "Type" merlin-imenu--type-list) index))
+    (when merlin-imenu--module-list
+      (push (cons "Module" merlin-imenu--module-list) index))
+    (when merlin-imenu--modtype-list
+      (push (cons "Modtype" merlin-imenu--modtype-list) index))
```

Comment:
If my above comment about `"Signature"` is correct, it might be worth using the same terminology here for consistency.

Reply:
Done, as well as the variable names.